components/camera/transformpipeline/classifier.go

```diff
@@ -52,6 +55,11 @@ func newClassificationsTransform(
 	if props.DistortionParams != nil {
 		cameraModel.Distortion = props.DistortionParams
 	}
+	validLabels := make(map[string]interface{})
+	for _, l := range conf.ValidLabels {
+		validLabels[strings.ToLower(l)] = struct{}{}
+	}
+	labelFilter := classification.NewLabelFilter(validLabels)
```

Comment:
The NewLabelFilter method is what'll check if validLabels is empty and not filter anything if it is (making it actually optional). I could check the len(validLabels) at this level but then I would have to sometimes pass a classifierSource with a labelFilter and sometimes not. I feel like that would just look ugly.